doc/releases/release-notes-3.3.rst

```diff
@@ -436,6 +436,30 @@ Drivers and Sensors
 
 * DMA
 
+  * STM32 DMA variable scope cleanups
+  * Intel GPDMA linked list transfer descriptors appropriately aligned to 64 byte addresses
+  * Intel GPDMA fix bug in transfer configuration to initialize cfg_hi and cfg_lo
```

Comment:
To be consistent with the rest of the release notes, please use the past tense. Also, adding a colon between the platform-driver name and the change description would improve readability.

```suggestion
  * Intel GPDMA: fixed bug in transfer configuration to initialize cfg_hi and cfg_lo
```

Reply:
Will fix this later myself.